src/services/AmazonS3Service.ts

```diff
@@ -0,0 +1,74 @@
+import axios, { AxiosError, AxiosResponse } from 'axios';
+
+import { Image } from '../types/image';
+
+const maxImageSize = 1000000;
```

Comment:
Can you add a comment or maybe change the variable name to include the units that this is in?